projects/js-toolkit/packages/liferay-cli/src/new/facet-localization/index.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+import {FilePath, TemplateRenderer, format} from '@liferay/js-toolkit-core';
+
+import prompt from '../util/prompt';
+
+import type {Facet, Options} from '../index';
+
+const {info, print} = format;
+
+const facet: Facet = {
+	async prompt(useDefaults: true, options: Options): Promise<Options> {
+		return await prompt(useDefaults, options, [
```

Comment:
Same with this prompt. Although I am probably missing something here...

Reply:
See comment above.